apm-lambda-extension/extension/apm_server_test.go

```diff
@@ -302,9 +382,21 @@ func TestContinuedAPMServerFailure(t *testing.T) {
 	pr, pw := io.Pipe()
 	gw, _ := gzip.NewWriterLevel(pw, gzip.BestSpeed)
 	go func() {
-		gw.Write([]byte(""))
-		gw.Close()
-		pw.Close()
+		_, err := gw.Write([]byte(""))
```

Comment:
for example,

```go
if _, err := gw.Write([]byte("")); err != nil {
  t.Fail()
  return
}
```

Reply:
Thanks for the advice ! In a34cd53 and 1bdfc14, I have used the shorthand wherever I could use it.